address-space-controller/src/main/java/io/enmasse/controller/TemplateInfraResourceFactory.java

```diff
@@ -100,6 +95,11 @@ private void prepareParameters(AddressSpace addressSpace,
             }
         }
         parameters.put(TemplateParameter.MESSAGING_SECRET, serviceCertMapping.get("messaging").getSecretName());
+
+        String brokerSupportUser =  String.format("broker-support-%s", UUID.randomUUID());
+        String brokerSupportPwd = UUID.randomUUID().toString();
```

Comment:
@robshelly I find prepareParameters could try to read the secret.  If it exists and username and password are populated pass those as args to the template.  otherwise, create the new defaults.